app/datatables/parent_object_datatable.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class ParentObjectDatatable < AjaxDatatablesRails::ActiveRecord
+  def view_columns
+    # Declare strings in this format: ModelName.column_name
+    # or in aliased_join_table.column_name format
+    @view_columns ||= {
```

Comment:
Future enhancement: Change the condition to match the field types more closely (numeric ids should probably be `:eq`)

Reply:
I played with this quite a bit - the 'like' is really nice on these because they filter down as you type it in, but it may not scale when we load a lot of records.